pkg/services/meta/notifications.go

```diff
@@ -398,12 +401,17 @@ func getFromMap(m *stackitem.Map, key string) stackitem.Item {
 	return m.Value().([]stackitem.MapElement)[i].Value
 }
 
-func (m *Meta) handleObjectNotification(s *containerStorage, e objEvent) error {
+func (m *Meta) handleObjectNotification(ctx context.Context, s *containerStorage, e objEvent) error {
 	if magic := uint32(e.network.Uint64()); magic != m.magicNumber {
 		return fmt.Errorf("wrong magic number %d, expected: %d", magic, m.magicNumber)
 	}
 
-	err := s.putObject(e)
+	h, err := m.net.Head(ctx, e.cID, e.oID)
+	if err != nil {
+		return fmt.Errorf("HEAD rpc error: %w", err)
```

Comment:
there is no guarantee that the RPC has been actually done - some preliminary errors are possible, or only local execution. Simply
```suggestion
		return fmt.Errorf("HEAD object: %w", err)
```

Reply:
well, changed but why?

Reply:
RPC is a remote communication, here we dont always go to remote